File: Altair.Core/Models/ParticipantModel.cs
namespace Altair.Core.Models
{
	/// <summary>
	/// Участник
	/// </summary>
	public class ParticipantModel
	{
		/// <summary>
		/// Идентификатор записи в базе данных
		/// </summary>
		public int Id { get; set; }

		/// <summary>
		/// Фамилия
		/// </summary>
		public string LastName { get; set; }

		/// <summary>
		/// Имя
		/// </summary>
		public string FirstName { get; set; }

		/// <summary>
		/// Отчество
		/// </summary>
		public string Patronymic { get; set; }

		/// <summary>
		/// Возраст
		/// </summary>
		public int Age { get; set; }

		/// <summary>
		/// Персональный код
		/// </summary>
		public string PersonalCode { get; set; }

		/// <summary>
		/// ФИО
		/// </summary>
		public string FullName => $"{LastName} {FirstName} {Patronymic}";
	}
}

File: Frontend.Vue/stars/src/models/participant.model.ts
import { IParticipantResponseModel } from '../response-models/participant.response-model.interface';

export class ParticipantModel {

    public lastName: string;
    public firstName: string;
    public patronymic: string;
    public age: number;
    public personalCode: string;

    constructor(dto: IParticipantResponseModel) {
        this.lastName = dto.lastName;
        this.firstName = dto.firstName;
        this.patronymic = dto.patronymic;
        this.age = dto.age;
        this.personalCode = dto.personalCode;
    }
}
